care/emr/tests/test_tag_config_api.py

```
@@ -0,0 +1,1671 @@
+            name="Test Organization", user=self.superuser
+        )
+        self.base_url = reverse("tag_config-list")
+        self.role = self.create_role_with_permissions(
+            permissions=[
+                TagConfigPermissions.can_read_tag_config.name,
+                TagConfigPermissions.can_write_tag_config.name,
+            ],
+        )
+
+    def generate_tag_config_data(
+        self, status=None, category=None, resource=None, slug=None, **kwargs
```

Ruff (0.13.2)

Unused method argument: `slug`

(ARG002)